test/correctness/allocator_alignment.cpp

```diff
@@ -0,0 +1,104 @@
+#include "Halide.h"
+#include "halide_benchmark.h"
+#include <stdio.h>
+
+using namespace Halide;
+using namespace Halide::ConciseCasts;
+
+namespace {
+
+void check(int r) {
+    assert(r == 0);
+}
+
+void run_test(Target t) {
+    // Must call this so that the changes in cached runtime are noticed
+    Halide::Internal::JITSharedRuntime::release_all();
```

Comment:
I'm surprised we don't invalidate the cache if the target changes.

Reply:
Me too. Want me to open an issue?

Reply:
https://github.com/halide/Halide/issues/7195